portfolio/src/main/webapp/style.css

```diff
@@ -45,6 +45,10 @@ body{
 
 }
 
+ul li ul li{
+  display: none;
+}
+
```

Comment:
Is it used anywhere? And in general, if you want to add style for some deep nested tags, it is easier to define a class for them.

Reply:
Sorry, I was going to add a drop-down for you to be able to access those pages through so I added that and a hover below but I ended up just using the learn more buttons.